packages/devextreme/js/__internal/core/utils/m_common.ts

```diff
@@ -7,6 +7,9 @@ import {
   isDefined, isFunction, isObject, isString, type,
 } from '@js/core/utils/type';
 
+// @ts-expect-error new deferred
+export const waitUntilFirstWidgetCreating = new Deferred();
+
```

Comment:
at this point it is assumed that will be only one instance of module but in module federation a module can have more than one instance...  won't this interfere?